CSETWebApi/CSETWeb_Api/CSETWeb_ApiCore/Controllers/AssessmentExportController.cs

```diff
@@ -61,5 +61,15 @@ public IActionResult ExportAssessment(string token)
 
             return File(result, "application/octet-stream", filename);
         }
+
+        [HttpPost]
+        [Route("api/assessment/makeDirectory")]
+        public IActionResult MakeDirectory([FromBody] string creatorName)
+        {
+            System.IO.Directory.CreateDirectory("C:\\\\Users\\mattr\\Downloads\\ExportAllTest");
```

Comment:
We should use something like this:
 `System.IO.Directory.CreateDirectory(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) + "\\Downloads\\ExportAllTest")`